bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -495,7 +590,24 @@ impl Client {
 
     /// Log out the current user
     pub fn logout(&self) -> Result<(), ClientError> {
-        RUNTIME.block_on(self.inner.matrix_auth().logout())?;
+        let Some(auth_api) = self.inner.auth_api() else {
```

Comment:
This feels super weird to me that we don't call `client.logout()` and have it direct the logout request appropriately, but as `client.logout()` was removed since I first started this branch, I've left the logic here.